uberfire-showcase/uberfire-client-webapp/pom.xml

```diff
@@ -1,292 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file supposed to be deleted too? Just double-checking.

Reply:
True it can stay, but also I did not find any locations that would use this. Best to keep it just in case.